[2024-03-05T20:54:38.718+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:38.723+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-05T20:54:38.724+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T20:54:38.730+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-05T20:54:38.733+0000] {standard_task_runner.py:60} INFO - Started process 315 to run task
[2024-03-05T20:54:38.735+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmp6dd218k5']
[2024-03-05T20:54:38.736+0000] {standard_task_runner.py:88} INFO - Job 29: Subtask download_dataset
[2024-03-05T20:54:38.760+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T20:54:38.784+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-03-05T20:54:38.800+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2024-03-05T20:54:38.801+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-05T20:54:38.802+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh 2023 12']
[2024-03-05T20:54:38.806+0000] {subprocess.py:86} INFO - Output:
[2024-03-05T20:54:38.808+0000] {subprocess.py:93} INFO - 
[2024-03-05T20:54:38.808+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2023/12
[2024-03-05T20:54:38.808+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2023/12
[2024-03-05T20:54:38.809+0000] {subprocess.py:93} INFO - /opt/***/include/download_bike_dataset.sh: 14: wget: Permission denied
[2024-03-05T20:54:38.809+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-03-05T20:54:38.814+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-03-05T20:54:38.816+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240305T205438, end_date=20240305T205438
[2024-03-05T20:54:38.821+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 29 for task download_dataset (Bash command failed. The command returned a non-zero exit code 1.; 315)
[2024-03-05T20:54:38.827+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T20:54:38.838+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-05T21:13:08.776+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-05T21:13:08.781+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-05T21:13:08.781+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-05T21:13:08.788+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-05T21:13:08.791+0000] {standard_task_runner.py:60} INFO - Started process 821 to run task
[2024-03-05T21:13:08.793+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmplc3ge8nc']
[2024-03-05T21:13:08.794+0000] {standard_task_runner.py:88} INFO - Job 59: Subtask download_dataset
[2024-03-05T21:13:08.819+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host 8277132cfd53
[2024-03-05T21:13:08.842+0000] {abstractoperator.py:707} ERROR - Exception rendering Jinja template for task 'download_dataset', field 'bash_command'. Template: "sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh {{ ds.strftime('%Y') }} {{ ds.strftime('%m') }}"
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:13:08.842+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2334, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2465, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2877, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/baseoperator.py", line 1241, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 699, in _do_render_template_fields
    rendered_content = self.render_template(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 174, in render_template
    return self._render(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/abstractoperator.py", line 654, in _render
    return super()._render(template, context, dag=dag)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/template/templater.py", line 131, in _render
    return render_template_to_string(template, context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 289, in render_template_to_string
    return render_template(template, cast(MutableMapping[str, Any], context), native=False)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/helpers.py", line 284, in render_template
    return "".join(nodes)
  File "<template>", line 13, in root
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 391, in call
    if not __self.is_safe_callable(__obj):
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/sandbox.py", line 275, in is_safe_callable
    getattr(obj, "unsafe_callable", False) or getattr(obj, "alters_data", False)
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 859, in __getattr__
    return self._fail_with_undefined_error()
  File "/home/airflow/.local/lib/python3.8/site-packages/jinja2/runtime.py", line 852, in _fail_with_undefined_error
    raise self._undefined_exception(self._undefined_message)
jinja2.exceptions.UndefinedError: 'str object' has no attribute 'strftime'
[2024-03-05T21:13:08.847+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240305T211308, end_date=20240305T211308
[2024-03-05T21:13:08.852+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 59 for task download_dataset ('str object' has no attribute 'strftime'; 821)
[2024-03-05T21:13:08.885+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-05T21:13:08.898+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T13:27:42.091+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:42.095+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-06T13:27:42.095+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T13:27:42.102+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-06T13:27:42.105+0000] {standard_task_runner.py:60} INFO - Started process 562 to run task
[2024-03-06T13:27:42.106+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpf8n6h5xl']
[2024-03-06T13:27:42.107+0000] {standard_task_runner.py:88} INFO - Job 86: Subtask download_dataset
[2024-03-06T13:27:42.130+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host 45431f0a53e1
[2024-03-06T13:27:42.158+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T13:27:42.175+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2024-03-06T13:27:42.176+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T13:27:42.176+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2024 01']
[2024-03-06T13:27:42.181+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T13:27:42.182+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:42.182+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2024/01
[2024-03-06T13:27:42.186+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T13:27:42.186+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T13:27:43.959+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  6 1839k    6  120k    0     0    97k      0  0:00:18  0:00:01  0:00:17   97k100 1839k  100 1839k    0     0  1037k      0  0:00:01  0:00:01 --:--:-- 1037k
[2024-03-06T13:27:43.961+0000] {subprocess.py:93} INFO - 
[2024-03-06T13:27:43.961+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T13:27:43.961+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T13:27:43.975+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240306T132742, end_date=20240306T132743
[2024-03-06T13:27:44.005+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T13:27:44.015+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-06T17:40:27.360+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-06T17:40:27.365+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-06T17:40:27.365+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-06T17:40:27.373+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-06T17:40:27.376+0000] {standard_task_runner.py:60} INFO - Started process 1481 to run task
[2024-03-06T17:40:27.378+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpiw7j1b44']
[2024-03-06T17:40:27.379+0000] {standard_task_runner.py:88} INFO - Job 121: Subtask download_dataset
[2024-03-06T17:40:27.405+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host e43ee8560afa
[2024-03-06T17:40:27.437+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-06T17:40:27.456+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2024-03-06T17:40:27.457+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-06T17:40:27.458+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2024 01']
[2024-03-06T17:40:27.463+0000] {subprocess.py:86} INFO - Output:
[2024-03-06T17:40:27.464+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:40:27.465+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2024/01
[2024-03-06T17:40:27.469+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-06T17:40:27.469+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-06T17:40:29.052+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 1839k    0  1533    0     0   2483      0  0:12:38 --:--:--  0:12:38  2480 42 1839k   42  781k    0     0   556k      0  0:00:03  0:00:01  0:00:02  556k100 1839k  100 1839k    0     0  1162k      0  0:00:01  0:00:01 --:--:-- 1161k
[2024-03-06T17:40:29.055+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv.zip
[2024-03-06T17:40:29.096+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv
[2024-03-06T17:40:29.096+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/__MACOSX/._JC-202401-citibike-tripdata.csv
[2024-03-06T17:40:29.268+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv (deflated 82%)
[2024-03-06T17:40:29.269+0000] {subprocess.py:93} INFO - 
[2024-03-06T17:40:29.269+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-06T17:40:29.270+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-06T17:40:29.284+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240306T174027, end_date=20240306T174029
[2024-03-06T17:40:29.316+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-06T17:40:29.330+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-07T14:09:26.762+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-07T14:09:26.768+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-07T14:09:26.768+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T14:09:26.775+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-07T14:09:26.778+0000] {standard_task_runner.py:60} INFO - Started process 1586 to run task
[2024-03-07T14:09:26.780+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmp1d23pc80']
[2024-03-07T14:09:26.781+0000] {standard_task_runner.py:88} INFO - Job 180: Subtask download_dataset
[2024-03-07T14:09:26.806+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host 0430cbdaac75
[2024-03-07T14:09:26.831+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T14:09:26.948+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2024-03-07T14:09:26.949+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T14:09:26.949+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2024 01']
[2024-03-07T14:09:26.956+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T14:09:26.957+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:09:26.957+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2024/01
[2024-03-07T14:09:26.961+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T14:09:26.961+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T14:09:28.600+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 1839k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 44 1839k   44  814k    0     0   556k      0  0:00:03  0:00:01  0:00:02  555k100 1839k  100 1839k    0     0  1122k      0  0:00:01  0:00:01 --:--:-- 1121k
[2024-03-07T14:09:28.602+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv.zip
[2024-03-07T14:09:28.642+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv
[2024-03-07T14:09:28.643+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/__MACOSX/._JC-202401-citibike-tripdata.csv
[2024-03-07T14:09:28.811+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv (deflated 82%)
[2024-03-07T14:09:28.812+0000] {subprocess.py:93} INFO - 
[2024-03-07T14:09:28.812+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T14:09:28.812+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv.zip
[2024-03-07T14:09:28.812+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T14:09:28.825+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240307T140926, end_date=20240307T140928
[2024-03-07T14:09:28.840+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T14:09:28.854+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-03-07T17:57:17.881+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-07T17:57:17.887+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [queued]>
[2024-03-07T17:57:17.887+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-07T17:57:17.896+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): download_dataset> on 2023-12-01 00:00:00+00:00
[2024-03-07T17:57:17.900+0000] {standard_task_runner.py:60} INFO - Started process 696 to run task
[2024-03-07T17:57:17.902+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'citibikedataset', 'download_dataset', 'scheduled__2023-12-01T00:00:00+00:00', '--job-id', '241', '--raw', '--subdir', 'DAGS_FOLDER/dag_citibikenyc.py', '--cfg-path', '/tmp/tmpk_4q18u1']
[2024-03-07T17:57:17.904+0000] {standard_task_runner.py:88} INFO - Job 241: Subtask download_dataset
[2024-03-07T17:57:17.929+0000] {task_command.py:423} INFO - Running <TaskInstance: citibikedataset.download_dataset scheduled__2023-12-01T00:00:00+00:00 [running]> on host a0c898e5643f
[2024-03-07T17:57:17.958+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2024-03-07T17:57:17.975+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='felipeparfitt' AIRFLOW_CTX_DAG_ID='citibikedataset' AIRFLOW_CTX_TASK_ID='download_dataset' AIRFLOW_CTX_EXECUTION_DATE='2023-12-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-01T00:00:00+00:00'
[2024-03-07T17:57:17.976+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-07T17:57:17.977+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sh ${AIRFLOW_HOME}/include/download_bike_dataset.sh         2024 01']
[2024-03-07T17:57:17.983+0000] {subprocess.py:86} INFO - Output:
[2024-03-07T17:57:17.985+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:57:17.985+0000] {subprocess.py:93} INFO - Downloading citibike dataset from 2024/01
[2024-03-07T17:57:17.989+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2024-03-07T17:57:17.989+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2024-03-07T17:57:20.687+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0 20 1839k   20  381k    0     0   162k      0  0:00:11  0:00:02  0:00:09  162k100 1839k  100 1839k    0     0   681k      0  0:00:02  0:00:02 --:--:--  681k
[2024-03-07T17:57:20.690+0000] {subprocess.py:93} INFO - Archive:  /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv.zip
[2024-03-07T17:57:20.730+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv
[2024-03-07T17:57:20.731+0000] {subprocess.py:93} INFO -   inflating: /opt/***/include/dataset/raw/2024/01/__MACOSX/._JC-202401-citibike-tripdata.csv
[2024-03-07T17:57:20.900+0000] {subprocess.py:93} INFO -   adding: opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv (deflated 82%)
[2024-03-07T17:57:20.901+0000] {subprocess.py:93} INFO - 
[2024-03-07T17:57:20.901+0000] {subprocess.py:93} INFO - Download completed successfully!
[2024-03-07T17:57:20.901+0000] {subprocess.py:93} INFO - /opt/***/include/dataset/raw/2024/01/JC-202401-citibike-tripdata.csv.zip
[2024-03-07T17:57:20.901+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-07T17:57:20.915+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=citibikedataset, task_id=download_dataset, execution_date=20231201T000000, start_date=20240307T175717, end_date=20240307T175720
[2024-03-07T17:57:20.927+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-07T17:57:20.940+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
